lib/jekyll/commands/serve.rb

```diff
@@ -45,7 +46,24 @@ def process(options)
             file_handler_options
           )
 
-          Jekyll.logger.info "Server address:", server_address(s, options)
+          server_address = server_address(s, options)
```

Comment:
this reassignment of `server_address` is confusing – could you rename the variable on the left here, please?